.run_cheri_qemu_and_test.py

```diff
@@ -26,7 +26,8 @@ def run_tests(qemu: boot_cheribsd.QemuCheriBSDInstance, args: argparse.Namespace
 
     # Run command on host to test the executed client
     os.chdir(f"{args.build_dir}/build")
-    subprocess.run(["/usr/bin/ctest", "-V", "--output-on-failure"], check=True)
+    subprocess.run(["../tests/prep_tests.sh"], check=True)
```

Comment:
I think it would be better to have this relative path wrt to `args.build_dir` ?

Reply:
Fixed in https://github.com/capablevms/CHERI-ELF-comp/pull/5/commits/9bffb24a00520d8ddf1966dd3a79b84a1298ff8d.